ports/esp32/machine_adc.c

```diff
@@ -63,6 +64,10 @@
     { MP_ROM_QSTR(MP_QSTR_ATTN_2_5DB), MP_ROM_INT(ADC_ATTEN_DB_2_5) }, \
     { MP_ROM_QSTR(MP_QSTR_ATTN_6DB), MP_ROM_INT(ADC_ATTEN_DB_6) }, \
     { MP_ROM_QSTR(MP_QSTR_ATTN_11DB), MP_ROM_INT(ADC_ATTEN_DB_11) }, \
+    { MP_ROM_QSTR(MP_QSTR_ATTN_MIN), MP_ROM_INT(ADC_ATTEN_MIN) }, \
+    { MP_ROM_QSTR(MP_QSTR_ATTN_MAX), MP_ROM_INT(ADC_ATTEN_MAX) }, \
+    { MP_ROM_QSTR(MP_QSTR_WIDTH_MIN), MP_ROM_INT(ADC_WIDTH_MIN) }, \
+    { MP_ROM_QSTR(MP_QSTR_WIDTH_MAX), MP_ROM_INT(ADC_WIDTH_MAX) }, \
```

Comment:
I don't think these min/max constants should be added.  The existing constants should be sufficient, and are explicit in the db/bit-width that they configure.